pyproject.toml

```diff
@@ -5,19 +5,7 @@ authors = [{ name = "PUDL", email = "pudl@catalyst.coop" }]
 requires-python = ">=3.12,<3.13"
 
 dependencies = [
-    "arelle-release>=2.29,<2.38",
-    "beautifulsoup4>=4.12.3,<5",
-    "catalystcoop.pudl @ git+https://github.com/catalyst-cooperative/pudl.git",
-    "coloredlogs>=14",
-    "dask>=2025,<2026",
-    "deltalake>=0.25,<1.1",
-    "feedparser>=6.0",
-    "frictionless>=5,<6",
-    "pydantic>=2.7,<3",
-    "python-dotenv~=1.1.0",
-    "pyyaml>=6.0,<7",
-    "semantic_version>=2.8,<3",
-    "tqdm>=4.64",
+    "catalystcoop.pudl @ git+https://github.com/catalyst-cooperative/pudl.git", "arelle-release>=2.37.23,<3", "beautifulsoup4>=4.13.4,<5", "coloredlogs>=14.0,<15", "dask>=2025.5.1,<2026", "deltalake>=1.0.2,<2", "feedparser>=6.0.11,<7", "frictionless>=5.18.1,<6", "pydantic>=2.11.7,<3", "python-dotenv>=1.1.0,<2", "pyyaml>=6.0.2,<7", "semantic-version>=2.10.0,<3", "tqdm>=4.67.1,<5",
```

Comment:
Is there some way that we can keep it from reformatting the dependencies like this? It's so much easier to see wtf is going on in diffs when there's just one per line.  Or use a TOML formatter pre-commit hook to enforce a particular format?